sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -467,6 +468,14 @@ def _stage_resources(self, options):
   def stage_file(self, gcs_or_local_path, file_name, stream,
                  mime_type='application/octet-stream'):
     """Stages a file at a GCS or local path with stream-supplied contents."""
+
```

Comment:
This may be platform-dependent. Will this WAI on Windows platform? 

See also: https://github.com/python/cpython/commit/29097d5a6aa13729583f98cd09923f97addb7819#diff-220640ea02a24dfdbb0647ef64f1a49bL24.

Reply:
I updated to use an alternative approach that should work on all platforms--PTAL.